src/explorer/model/Dependency.ts

```diff
@@ -16,15 +16,17 @@ export class Dependency extends TreeNode implements ITreeItem {
     private _scope: string;
     private _omittedStatus: IOmittedStatus;
     private _uri: vscode.Uri;
-    constructor(gid: string, aid: string, version: string, scope: string, omittedStatus: IOmittedStatus, projectPomPath: string) {
+    constructor(gid: string, aid: string, version: string, scope: string, projectPomPath: string, omittedStatus?: IOmittedStatus) {
         super();
         this._gid = gid;
```

Comment:
what's the point to keep private member `_var` with a public getter `var`? e.g. scope, version, etc.

It would be natural and simple to make them public.